src/config/entities/__tests__/configuration.ts

```diff
@@ -285,6 +286,56 @@ export default (): ReturnType<typeof configuration> => ({
       81457: faker.string.hexadecimal({ length: 32 }),
       11155111: faker.string.hexadecimal({ length: 32 }),
     },
+    noFeeCampaign: {
+      1: {
+        startsAtTimeStamp: new Date().getTime() / 1000 - 10_000,
+        endsAtTimeStamp: new Date().getTime() / 1000 + 1000_000,
+        safeTokenAddress: faker.finance.ethereumAddress(),
+        maxTxValueInUSD: 100,
```

Comment:
We shouldn't hardcode the number so that different cases are tested:

```suggestion
        maxTxValueInUSD: faker.number.int(),
```